clj/src/slipstream/ui/models/vms.clj

```diff
@@ -1,30 +1,42 @@
 (ns slipstream.ui.models.vms
   (:require [superstring.core :as s]
             [net.cgrand.enlive-html :as html]
-            [slipstream.ui.util.clojure :as uc]
+            [slipstream.ui.util.time :as time]
+            [slipstream.ui.util.current-user :as current-user]
             [slipstream.ui.models.pagination :as pagination]))
 
 (defn- parse-vm
   [vm-metadata]
-  (-> vm-metadata
-      (select-keys [:state
-                    :measurement])
-      (assoc        :username           (-> vm-metadata :user       not-empty))
-      (assoc        :cloud-instance-id  (-> vm-metadata :instanceId not-empty))
-      (assoc        :run-uuid           (some-> vm-metadata :runUuid (s/replace "Unknown" "") not-empty))
-      (assoc        :cloud-name         (-> vm-metadata :cloud      not-empty))
-      (assoc        :ip-address         (-> vm-metadata :ip         not-empty))
-      (assoc        :run-owner          (-> vm-metadata :runOwner   not-empty))
-      (assoc        :name               (-> vm-metadata :name       not-empty))
-      (assoc        :cpu                (-> vm-metadata :cpu        not-empty))
-      (assoc        :ram                (-> vm-metadata :ram        not-empty))
-      (assoc        :disk               (-> vm-metadata :disk       not-empty))
-      (assoc        :instance-type      (-> vm-metadata :instanceType not-empty))))
+  (let [run-uuid   (some-> vm-metadata :runUuid (s/replace "Unknown" "") not-empty)
+        run-owner  (-> vm-metadata :runOwner not-empty)
+        recent-vm? (-> vm-metadata :measurement time/parse time/recent?)]
+    (-> vm-metadata
+        (select-keys [:state
+                      :measurement])
```

Comment:
It's part of the model, so it's there. Currently it's not exposed in the UI indeed.